minigalaxy/installer.py

```diff
@@ -167,6 +170,35 @@ def copy_thumbnail(game):
     return error_message
 
 
+def create_applications_file(game):
+    error_message = ""
+    preferences_switch = Config.get("create_applications_file")
+    if game.platform == "linux" and preferences_switch:
```

Comment:
We can easily support more platforms. Please remove this Linux check to:
"
    if Config.get("create_applications_file"):
"
(lines 175 and 176 combined into one)

Reply:
Done!